### DISCUSSION TITLE
关于音源参数的问题

### DISCUSSION THREAD

**Discussion by Syozhi-1** [qier222/YesPlayMusic#1331] [Q&A] [answered]
看了UNM那边说有参数，没教如何获取参数，也没教怎么使用参数；
例如migu的音源，MIGU_COOKIE="<your_aversionid>" 又不教怎么用，客户端也不说怎么用，就说个逗号隔开各音源，之前提了个issue，说加参就行，我拿到了我的migu cookie，但是加哪？怎么加？
请求写个readme告知一下音源参数的获取和使用，十分感谢！

![image](https://user-images.githubusercontent.com/58074198/155055231-9c93982b-ddd7-47c1-b807-3abc2644829e.png)
是这样使用吗？

**Comment by pan93412** [comment, marked as answer]
那個 MIGU_COOKIE 是要加進「環境變數」裡頭：「鍵 (key) 」是 `MIGU_COOKIE`；「值 (value)」是你的 Migu Cookie。

未來我們會改善這方面的使用者體驗，會直接把這部分設定直接納入 YPM 的設定介面。抱歉讓您帶來不便。
